bundle/LegacyMapper/SiteAccess.php

```diff
@@ -90,6 +90,24 @@ public function onBuildKernelWebHandler( PreBuildKernelWebHandlerEvent $event )
             $uriPart = array_diff( $aPathinfo, $aSemanticPathinfo, $this->getFragmentPathItems() );
         }
 
+        if ($siteAccess->matchingType == 'compound:logicalAnd(Map\Host, Map\URI)')
```

Comment:
You're hardcoding the matching type check here. What if it was in the reverse order? Or what if `URIElement` was used instead of `Map\URI`?